src/tests/ftest/util/server_utils.py

```diff
@@ -511,7 +511,7 @@ def start(self):
         self.log.info("<SERVER> Formatting hosts: <%s>", self.dmg.hostlist)
         # Temporarily increasing timeout to avoid CI errors until DAOS-5764 can
         # be further investigated.
-        self.dmg.storage_format(timeout=40)
+        self.dmg.storage_format(timeout=self.format_timeout)
```

Comment:
(pylint-no-member) Instance of 'DaosServerManager' has no 'format_timeout' member